src/content/components/common/follow.js

```diff
@@ -68,7 +69,7 @@ export default class Follow {
     return browser.runtime.sendMessage({
       type: messages.OPEN_URL,
       url: element.href,
-      newTab: this.newTab,
+      newTab: true,
```

Comment:
This change allows to open a new tab even when newTab is false.
I think it's better to prioritize user's newTab option.